main.go

```diff
@@ -91,12 +92,23 @@ func main() {
 		tableFlag       = flag.StringP("table", "t", "TABLE", "table name of the sql format")
 		templateFlag    = flag.StringP("template", "T", "", "Use template as input")
 	)
+	flag.StringVarP(&completionFlag, "completion", "C", "", "print bash/zsh completion function, pass shell as argument (\"bash\" or \"zsh\")")
+
```

Comment:
if we use `flag.StringP`  then all the declaration look consistent. As far as I can say, they're equivalent forms?